### PR TITLE
[Style] #2349 통합 대시보드 과업 우선 재구성 (PR⑦)

### PR DESCRIPTION
**Pull request by AquilaXk**
## 관련 이슈

Refs #2349

## 작업 내용

- "확인 필요" 통합 카드 신설: 확인할 제보·확인 필요 시설·데이터팩 차단 요인을 페이지 제목 바로 아래 하나의 카드로 모으고, 각 항목은 기존 뷰 모델 값을 그대로 재사용해 해당 화면(제보 확인 대기열/시설 상태판)이나 데이터팩 출시 준비 섹션 앵커로 딥링크한다. 0건 항목은 행 자체를 생략하고, 전부 0(또는 권한 밖)이면 카드 대신 `.admin-status.good` 한 줄 상태 텍스트("지금 확인할 항목이 없습니다")로 대체한다.
- 위 카드가 흡수한 중복·약한 섹션 제거: 상단 알림 바의 데이터팩 blocker 중복 서술, "먼저 확인할 제보"(안내문+버튼뿐이던 섹션)를 제거했다.
- 데스크톱 2열 그리드: 하단 "데이터팩 출시 준비"와 "운영 이상 신호"를 `admin-grid-2`로 나란히 배치해 우측 부유를 해소했다(데이터팩 권한이 없으면 운영 이상 신호가 단독 폭으로 렌더).
- 차단 요인 상세 표 노이즈 제거: 후보 게이트·별칭·격리·수동 오버라이드·시설 근거·경로 게이트·매니페스트 서명 7개 카테고리 중 0건은 숨기고 비-0만 노출, 하단에 숨긴 건수 요약 줄을 추가했다. 전부 0건이면 표 대신 기존 빈 상태 fragment(#2327 문법)를 적용한다.
- 후보명 결측 시 노출되던 `-` 원문("- ● 확인 필요")을 결측 규칙에 맞춰 `—`로 정리했다.
- 뷰포트 적응: 태블릿(1024/768)은 `admin-grid-2`의 기존 반응형 규칙으로 1열 전환, 모바일은 HTML 순서(확인 필요 카드 → 핵심 지표 → 추이 → 상세 섹션)로 자연스럽게 단열 순서를 충족한다.
- flash message는 기존 `admin/fragments/shell :: flash` 컴포넌트(`.alert`)를 이미 사용 중임을 실기동으로 확인했다(변경 없음).
- 지표 집계·조회 로직, backend API, 기존 링크 URL은 변경하지 않았다(`AdminOverviewPageController` 뷰 모델 가공과 템플릿·CSS만).

## 검증

- 실행한 명령과 결과:
  - `cd backend && ./gradlew compileJava` — 성공
  - `cd backend && ./gradlew test --tests "com.easysubway.admin.*" --tests "com.easysubway.datapack.*" --tests "com.easysubway.report.*"` — 전부 통과(AdminDashboardPageTest, AdminV3PageSmokeTest, AdminDesignGuardTest, DatapackReleaseBlockerSummaryAdminPageTest 포함)
  - `cd backend && ./gradlew test --tests "com.easysubway.admin.adapter.in.web.AdminDashboardTriagePanelTest"` — 신설 테스트 5건 통과(0건 생략, 전부 0 폴백, 차단 요인 상세 0건 숨김·요약, 전부 0 빈 상태, 후보명 em dash 표기)
  - `node --test tools/qa/admin-accessibility-qa.test.mjs` — 15/15 통과
  - dev 실기동(H2, `EASYSUBWAY_DEV_SEED=true`/미설정 각각 별도 인메모리 인스턴스, 임의 포트, 종료 후 정리): 1440/1024/390 스크린샷을 로컬 전용 `.superpowers/plans/2349-evidence/`에 저장(리포에는 커밋하지 않음)

## 영향

- [x] 제품/운영 위험 없음 (route/accessibility/mobile UX/backend API/auth/security 아님)
- [x] DB migration 없음
- [x] 배포 영향 없음
- [x] route/realtime/datapack contract 영향 없음
- [x] CI workflow·계약 테스트·release gate JSON 변경 없음 (있으면 full.md로 전환)

## 체크리스트

- [x] 이 PR은 B/C등급 작업이며 full template이 필요 없다.
- [ ] CI 결과를 확인했다.
- [ ] GitHub PR Review 객체가 있는지 확인했다. CodeRabbit status check만으로는 리뷰 완료로 보지 않는다.
- [ ] CodeRabbit 실행이 불가능하거나 PR Review 객체가 없으면 폴백 리뷰를 단일 PR review로 게시했다.

## Summary by CodeRabbit

* **개선 사항**
  * 관리자 대시보드에 통합 “확인 필요” 카드가 추가되었습니다.
  * 데이터팩 출시 준비 현황을 권한에 따라 표시하고, 차단 요인 상세 내역을 확인할 수 있습니다.
  * 차단 요인이 없는 항목은 자동으로 숨겨져 핵심 정보만 간결하게 표시됩니다.
  * 데이터가 없을 때 안내 문구와 빈 상태 화면을 제공합니다.
  * 긴급 신호에서 데이터팩 차단 항목을 분리해 더 명확하게 확인할 수 있습니다.
  * 후보 정보가 없을 경우 대시(—)로 일관되게 표시됩니다.

### CODE REVIEW

File: backend/src/main/resources/templates/admin/dashboard.html [hunk 1]
```diff
@@ -85,60 +106,70 @@ <h2 id="dashboard-metric-panel-title">핵심 지표</h2>
...
+						<td th:text="${row.count}">0</td>
+					</tr>
+					</tbody>
+				</table>
+				</div>
+				</th:block>
+				<p class="summary" th:if="${datapackHiddenBlockerRowCount > 0 and !datapackBlockerRows.isEmpty()}"
+				   th:text="|그 외 항목 ${datapackHiddenBlockerRowCount}건은 0건입니다|">그 외 항목 0건</p>
+				<th:block th:if="${datapackBlockerRows.isEmpty()}">
+				<div th:replace="~{admin/fragments/empty-state :: panel('차단 요인이 없습니다.', null)}"></div>
```
Comment (AquilaXk) [inline backend/src/main/resources/templates/admin/dashboard.html]: _🎯 Functional Correctness_ | _🟠 Major_ | _⚡ Quick win_

**차단 요인 상세 표·빈 상태가 `totalBlockers`와 불일치 — 같은 화면에서 "1건 있음"과 "없습니다"가 동시 노출될 수 있습니다**

상세 표(`datapackBlockerRows`)와 "그 외 N건" 요약은 7개 카테고리만 다루지만, 트리아지 카드와 `blocker-total`이 쓰는 `totalBlockers`는 `sourceFreshnessBlockers`·`callbackReconciliationBlockers`·evidence bundle까지 합산합니다. 원천 스냅샷 만료 1건 + 표시 7개 카테고리 전부 0인 상태면 트리아지 카드는 "데이터팩 차단 요인 1건", 상세 표는 "차단 요인이 없습니다." 빈 상태를 렌더해 직접 모순됩니다 — 기존 7행 상시 나열에는 없던 거짓 단정을 이번 empty-state가 새로 도입합니다. 혼합 시나리오에서 "그 외 항목 N건은 0건입니다" 문구도 `blocker-total`과 어긋나 보입니다.

수정 방향: 상세 표에 나머지 구성요소 행(원천 신선도·콜백 정합·증거 번들)을 추가해 표 합 == `totalBlockers`가 되게 하고 empty-state·"그 외" 계산도 같은 집합 기준으로 맞추거나, 최소한 empty-state를 `totalBlockers == 0`일 때만 렌더하고 그 외에는 미표시 차단 요인 유도 문구로 대체하세요.

<details>
<summary>🤖 Prompt for AI Agents</summary>

```
Verify each finding against current code. Fix only still-valid issues, skip the
rest with a brief reason, keep changes minimal, and validate.

In `@backend/src/main/resources/templates/admin/dashboard.html` around lines 152 - 170 (and the row-building logic in AdminOverviewPageController), make the blocker detail table, hidden-count summary, and empty-state consistent with `datapackReleaseSummary.totalBlockers` — either add rows for source freshness, callback reconciliation, and evidence bundle blockers so the table sums to totalBlockers, or gate the empty-state on totalBlockers == 0 with a fallback message otherwise.
```

</details>

_Source: Claude Code review_

File: backend/src/test/java/com/easysubway/admin/adapter/in/web/AdminDashboardTriagePanelTest.java [hunk 2]
```diff
@@ -0,0 +1,217 @@
...
+			0,
+			manualOverride,
+			facility,
+			routeGate,
+			manifest,
+			List.of(),
+			null
+		);
+	}
+}
```
Comment (AquilaXk) [inline backend/src/test/java/com/easysubway/admin/adapter/in/web/AdminDashboardTriagePanelTest.java:263]: _📐 Maintainability & Code Quality_ | _🟡 Minor_ | _⚡ Quick win_

**테스트 헬퍼가 `totalBlockers > 7행 합` 시나리오를 재현할 수 없어 위 모순 케이스를 구조적으로 미포착합니다**

`blockerSummary(...)` 헬퍼가 total을 7개 인자 합으로만 만들고 `sourceFreshnessBlockers`를 0으로 고정해, "전부 0 → empty-state" 테스트가 실제로는 `totalBlockers == 0`만 검증합니다. `totalBlockers > 0`인데 표시 7행이 전부 0인 모순 케이스는 검증 범위 밖입니다. 또한 `.contains("확인할 제보")` 라벨 assert는 렌더된 안내 주석과도 매치되어 실질 고정력이 약합니다.

수정 방향: 헬퍼에 total(또는 비표시 구성요소)을 분리 주입하는 인자를 추가해 모순 케이스 회귀 테스트를 넣고, 라벨 assert는 태그 스코핑(`dashboard-triage-label` 요소 기준)으로 강화하세요. 트리아지 앵커(`#dashboard-datapack-readiness`)가 실제 href로 출력되는 positive assert도 함께 고정하면 좋습니다.

<details>
<summary>🤖 Prompt for AI Agents</summary>

```
Verify each finding against current code. Fix only still-valid issues, skip the
rest with a brief reason, keep changes minimal, and validate.

In `@backend/src/test/java/com/easysubway/admin/adapter/in/web/AdminDashboardTriagePanelTest.java` around lines 180 - 217, let the blockerSummary helper inject totalBlockers independently of the seven visible categories, add a regression test for totalBlockers > 0 with all visible rows zero, scope label assertions to the rendered triage label markup, and positively assert the `#dashboard-datapack-readiness` anchor href.
```

</details>

_Source: Claude Code review_

**Review by AquilaXk — COMMENTED** [review]
**Actionable comments posted: 2**

<details>
<summary>🤖 Prompt for all review comments with AI agents</summary>

```
Verify each finding against current code. Fix only still-valid issues, skip the
rest with a brief reason, keep changes minimal, and validate.

Inline comments:
In `@backend/src/main/resources/templates/admin/dashboard.html` around lines 152 - 170, make the blocker detail table, hidden-count summary, and empty-state consistent with `totalBlockers` — add rows for source freshness, callback reconciliation, and evidence bundle blockers so the table sums to totalBlockers, or gate the empty-state on totalBlockers == 0.
In `@backend/src/test/java/com/easysubway/admin/adapter/in/web/AdminDashboardTriagePanelTest.java` around lines 180 - 217, allow totalBlockers to be injected independently, add a regression test for totalBlockers > 0 with all visible rows zero, scope label assertions to rendered markup, and positively assert the datapack readiness anchor href.

---

Outside diff comments:
None.
```

</details>

<details>
<summary>🪄 Autofix (Beta)</summary>

Claude Code fallback에는 CodeRabbit Autofix가 없습니다. 각 finding을 현재 코드에서 검증한 뒤 별도 승인된 수정 흐름으로 처리합니다.

</details>

---

<details>
<summary>ℹ️ Review info</summary>

<details>
<summary>⚙️ Run configuration</summary>

**Configuration used**: Path: .coderabbit.yaml

**Review profile**: ASSERTIVE

**Plan**: Claude Code fallback

**Run ID**: `N/A (Claude Code fallback)`

**Review source**: Claude Code (독립 reviewer subagent, Opus)

**Repository visibility**: public GitHub repository

**Trigger**: CodeRabbit status check만 생성되고 Review 객체 미생성 확인 후 즉시 폴백

**Base**: `main @ 92bf61187a6c7b418065ca100d7e24ff33db5949`

**Head**: `refactor/2349-dashboard @ 6186b41c9f59ebb7b8c91f7124efca48b103e1f7`

**Command**: `gh pr diff 2352 --repo AquilaXk/easysubway`

</details>

<details>
<summary>📥 Commits</summary>

Reviewing files that changed from the base of the PR and between 92bf61187a6c7b418065ca100d7e24ff33db5949 and 6186b41c9f59ebb7b8c91f7124efca48b103e1f7.

</details>

<details>
<summary>📒 Files selected for processing (4)</summary>

* `backend/src/main/java/com/easysubway/admin/adapter/in/web/AdminOverviewPageController.java`
* `backend/src/main/resources/static/css/admin-data.css`
* `backend/src/main/resources/templates/admin/dashboard.html`
* `backend/src/test/java/com/easysubway/admin/adapter/in/web/AdminDashboardTriagePanelTest.java`

</details>

</details>

<details>
<summary>📜 Review details</summary>

<details>
<summary>🧰 Additional context used</summary>

<details>
<summary>📓 Path-based instructions (1)</summary>

<details>
<summary>backend/**</summary>

**⚙️ CodeRabbit configuration file**

> 인증, 보안, 트랜잭션, 입력 검증, 데이터 일관성, 전자정부프레임워크 경계 위반을 우선 확인한다.

Files:
- `backend/src/main/java/com/easysubway/admin/adapter/in/web/AdminOverviewPageController.java`
- `backend/src/main/resources/static/css/admin-data.css`
- `backend/src/main/resources/templates/admin/dashboard.html`
- `backend/src/test/java/com/easysubway/admin/adapter/in/web/AdminDashboardTriagePanelTest.java`

</details>

</details>

</details>

</details>

File: backend/src/main/resources/templates/admin/dashboard.html (hunk 1 above)
Comment (AquilaXk) [reply backend/src/main/resources/templates/admin/dashboard.html]: 커밋 64eaa4622980b26a8e868d8e5b93352045ce7a6d에서 수정했습니다.

- `DatapackReleaseBlockerSummary`에 `callbackReconciliationBlockers`·`evidenceBundleBlockers`(검증기/경로 회귀/Android 증거, 매니페스트 서명 제외분) 필드를 추가해 저장소 집계값을 뷰 모델에 노출
- `AdminOverviewPageController`의 상세 표를 7개에서 10개 행(소스 최신성·콜백 정합성 확인·증거 번들 검증 추가)으로 확장해 표 합계가 항상 `totalBlockers`와 일치하도록 수정("증거 번들"은 위 sha·워크플로 표의 동일 라벨 행과 충돌해 "증거 번들 검증"으로 표기)

검증: `./gradlew test --tests "com.easysubway.admin.*" --tests "com.easysubway.datapack.*"` 통과, `node --test tools/qa/admin-accessibility-qa.test.mjs` 통과(15/15), CI 전 체크 green.

File: backend/src/test/java/com/easysubway/admin/adapter/in/web/AdminDashboardTriagePanelTest.java (hunk 2 above)
Comment (AquilaXk) [reply backend/src/test/java/com/easysubway/admin/adapter/in/web/AdminDashboardTriagePanelTest.java:263]: 커밋 64eaa4622980b26a8e868d8e5b93352045ce7a6d에서 수정했습니다.

`blockerSummary` 헬퍼를 재설계해 `total`을 구성 요소 합과 분리 주입 가능하게 했고(11개 인자: total + candidateGate/alias/quarantine/sourceFreshness/manualOverride/facility/routeGate/callbackReconciliation/evidenceBundle/manifest), "totalBlockers>0인데 표시 행 전부 0" 모순 회귀 테스트(`datapackBlockerDetailExposesPreviouslyUncountedComponents`)를 추가해 수정 후 소스 최신성·콜백 정합성 확인·증거 번들 검증 행이 노출됨을 assert했습니다. 라벨 assert는 `dashboard-triage-label` 마크업으로 스코핑을 강화했고, 같은 테스트에서 트리아지 앵커 `href="#dashboard-datapack-readiness"` positive assert도 추가했습니다.

검증: `./gradlew test --tests "com.easysubway.admin.*" --tests "com.easysubway.datapack.*"` 통과, CI 전 체크 green.
